src/Tools/GenerateSdkPackages/make-all.ps1

```diff
@@ -0,0 +1,65 @@
+Param(
+    [string]$version = "26127.00",
+    [string]$branch = "d15rel",
+    [string]$outPath = $null,
+    [string]$fakeSign = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try {
+    if ($outPath -eq "") {
```

Comment:
There's no way to tell powershell "we're running non-interactively, if you hit a mandatory parameter crash instead?"